proposals/nnnn-alert-style-subtle.md

```diff
@@ -0,0 +1,132 @@
+
+#### Alert
+```xml
+<enum name="AlertStyle" since="X.Y">
+    <description>How the alert is laid out on screen</description>
+    <element name="PROMINENT">
+        <description>A modal alert style that requires interaction, blocking interaction with any other part of the display.</description>
+    </element>
+    <element name="SUBTLE">
+        <description>A new alert style like a notification banner that doesn't take over the entire display and allows interaction with the current app. e.g. A navigation alert that lets the user know of an upcoming blockage on the road and asking the user if they wish to reroute.</description>
+    </element>
+</enum>
+
+<function name="Alert" functionID="AlertID" messagetype="request">
+    <description>Shows an alert which typically consists of text-to-speech message and text on the display. At least either alertText1, alertText2 or TTSChunks need to be provided.</description>
+    
+    ...
+    <!-- Additions -->
+    <param name="style" type="AlertStyle" defvalue="PROMINENT" mandatory="false" since="X.Y">
+        <description>Sets the layout of the alert screen.</description>
+    </param>
+</function>
+```
+
+Also note that if [Cancel Interaction](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0184-cancel-interaction.md) is implemented, this would be affected as well.
+
+### Mockup
+![Subtle Alert Mockup](../assets/proposals/nnnn-alert-style-subtle/subtle-alert-mockup.png)
+
+## Potential downsides
+Because this is an addition to the `Alert` RPC and not a new separate RPC, policies cannot allow an app to send a "subtle" notification while preventing them from sending a "prominent" notification. This is a significant downside and almost caused the author to change the approach. The author ultimately determined that adding a new non-mandatory parameter was better due to the API similarity, but if the SDLC decides to use Alternative 1 below, that would be acceptable by the author.
+
+## Impact on existing code
+This would be a minor version change.
+
+## Alternatives considered
+
+### Alternative 1: Separate RPC
+Using a separate RPC allows us to (1) control this alert style separately from `Alert` in policies, and (2) to provide a callback for when the notification is touched. This would permit better control for the app developers than simply opening the app.
+
+```xml
+<function name="SubtleAlert" functionID="SubtleAlertID" messagetype="request">
+    <description>Shows an alert which typically consists of text-to-speech message and text on the display. At least either alertText1, alertText2 or TTSChunks need to be provided.</description>
+    
+    <description>Shows an alert which typically consists of text-to-speech message and text on the display. At least either alertText1, alertText2 or TTSChunks need to be provided.</description>
```

Comment:
Is this description supposed to be here twice?

Reply:
🤔 I don't know how that happened